src/controllers/handlers/scene-admin-handlers/list-scene-admins-handler.ts

```diff
@@ -69,18 +72,40 @@ export async function listSceneAdminsHandler(
 
   const admins = await sceneAdminManager.listActiveAdmins(sceneAdminFilters)
 
-  let adminsNames: Record<string, string>
-  if (admins.length > 0) {
-    const adminsWallets = admins.map((admin) => admin.admin)
-    adminsNames = await names.getNamesFromAddresses(adminsWallets)
+  const extraAddresses = new Set<string>()
+  let worldActionPermissions: PermissionsOverWorld | undefined
+
+  isWorlds && (worldActionPermissions = await fetchWorldActionPermissions(place.world_name!))
+
+  if (worldActionPermissions?.deployment.type === PermissionType.AllowList) {
+    worldActionPermissions.deployment.wallets.forEach((wallet) => extraAddresses.add(wallet.toLowerCase()))
+  }
+
+  if (worldActionPermissions?.streaming.type === PermissionType.AllowList) {
+    worldActionPermissions.streaming.wallets.forEach((wallet) => extraAddresses.add(wallet.toLowerCase()))
```

Comment:
We could prevent additional checks (`.?`) by moving these two conditionals within the prior one: `if (worlds)`

wdyt?

Reply:
I prefer to avoid nested code, if that's ok with you

Reply:
Fine by me if we assume that this code will mean more conditionals to check by the resulting code.